en/middleware/micrortps.md

```diff
@@ -0,0 +1,338 @@
+# PX4-FastRTPS bridge
+
+
+This bridge adds communication capabilities between a **PX4 Autopilot** and a **Fast RTPS** application through serial ports or
+UDP sockets using **CDR serialization**. The goal is to provide a DDS (Data Distribution Service) interface to PX4. This interface allows sharing information also with the upcoming release of ROS (Robot Operating System), ROS2.
```

Comment:
Would be good to turn **CDR serialization** into a link. I don't know what this term means.